src/python/services/queue-service/queue_service.py

```diff
@@ -0,0 +1,63 @@
+from azure.identity import AzureCliCredential
+from azure.storage.queue import QueueServiceClient
+from azure.ai.formrecognizer import FormRecognizerClient
+from azure.ai.textanalytics import TextAnalyticsClient
+from azure.cosmos import CosmosClient
+
+import requests
+import os
+import time
+import json
+
+from dotmap import DotMap
+from dotenv import load_dotenv
+
+load_dotenv("../../.env")
+
+credential = AzureCliCredential()
+
+queue_service_client = QueueServiceClient(
+    account_url=os.getenv("AZURE_STORAGE_QUEUE_ENDPOINT"), credential=credential
```

Comment:
By using os.getenv, there is an expectation of `None` being returned if the env key doesn't exist. Right now, that case isn't handled.

Reply:
The SDK client will except if None is passed in, I think I'm okay with that as env vars are documented.   I would like to keep this simple and not validate a bunch of config with custom exception messages, if prod app, yes.